corehq/apps/users/models.py

```diff
@@ -2772,3 +2773,74 @@ def can_view_web_uers(self):
 
     def can_view_roles(self):
         return False
+
+
+class UserReportingMetadataStaging(models.Model):
+    domain = models.TextField()
+    user_id = models.TextField()
+    app_id = models.TextField()
```

Comment:
It looks like system [forms such as this](https://www.icds-cas.gov.in/a/icds-cas/reports/form_data/cbfa0d3963f241a9b5c6cbd53b355e13/#form-xml) could have null app_id, so these are resulting in ton of [pillow errors](https://www.icds-cas.gov.in/admin/pillow_retry/pillowerror/?error_type=django.db.utils.IntegrityError&pillow=xform-pillow)

Reply:
Fixed here https://github.com/dimagi/commcare-hq/pull/25923